xgcm/grid.py

```diff
@@ -1111,6 +1152,36 @@ def derivative(self, da, axis, **kwargs):
         dx = self.get_metric(diff, ("X",))
         return diff / dx
 
+    @docstrings.dedent
+    def integrate(self, da, axis, **kwargs):
```

Comment:
what if I want to integrate over multiple dimensions at once, e.g. ('X', 'Y')

Reply:
That works, and is tested in `test_integrate()` in the test module. I however only use lists like `['X','Y']`. Should I add tuples?